package.json

```diff
@@ -68,6 +68,7 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^5.1.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
```

Comment:
Can you move this package to [boilerplate/package.json](https://github.com/infinitered/ignite/blob/master/boilerplate/package.json) instead? 

This top level package.json is for the cli, where as the `boilerplate` folder is for the app content.